source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,56 @@
+RoboRIO2 Imaging
+================
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the SD card -- with the roboRIO using the imaging tool, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.
```

Comment:
This seems either incomplete or superfluous. I'd recommend a minimum size and and perhaps some recommendation on replacements

Reply:
Agreed this was left incomplete and noted [here](https://usfirst.collab.net/sf/discussion/do/listPosts/projects.frc_2021_beta/discussion.ni_utilities.topc2224)